pkg/support/support_test.go

```diff
@@ -347,7 +347,7 @@ func TestSupportBundleBuilder_GenerateSupportBundle(t *testing.T) {
 			config := mocks.NewBundleConfig(ctrl)
 			finch := fpath.Finch("mockfinch")
 			ecc := mocks.NewCommandCreator(ctrl)
-			lcc := mocks.NewLimaCmdCreator(ctrl)
+			ncc := mocks.NewLimaCmdCreator(ctrl)
```

Comment:
saw that `mocks.NewLimaCmdCreator` has been renamed to `mocks.NewNerdctlCmdCreator` in pkg/mocks/command_nerdctl_cmd_creator.go